src/server/services/userService.ts

```diff
@@ -12,7 +12,7 @@ export const registrationSchema = z.object({
 export type CreateUserInputType = z.infer<typeof registrationSchema>;
 
 export const createUser = async (user: CreateUserInputType) => {
-  if (user.type === "CUSTOMER") {
+  if (user.type === UserType.CUSTOMER) {
```

Comment:
So looking at this again, I am seeing that `UserType` is not an enum it's a type. Prisma compiles it to:
```ts
export const UserType: {
  SHOP_OWNER: 'SHOP_OWNER',
  EMPLOYEE: 'EMPLOYEE',
  CUSTOMER: 'CUSTOMER'
};

export type UserType = (typeof UserType)[keyof typeof UserType]
```

I think we should leave it as a string constant and let TypeScript's static analysis take care of checking if it is a valid statement. I would prefer not to create our own enum in a separate types file and have to manage it in two different places.